recipes/jupyter_qtconsole_colorschemes/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright (C) 2014 Joon Ro
```

Comment:
Would raise an issue with upstream about getting this included in the `MANIFEST.in`. Seems like this is their [issue tracker]( https://bitbucket.org/joon/color-schemes-for-jupyter-qt-console/issues?status=new&status=open ).